cores/esp8266/FS.h

```diff
@@ -125,6 +125,9 @@ class FS
     bool rename(const char* pathFrom, const char* pathTo);
     bool rename(const String& pathFrom, const String& pathTo);
 
+    bool gc(uint32_t size);
+    bool gc_quick();
```

Comment:
These are SPIFFS specific functions, I don't think they should be present in the generic FS interface.

Reply:
Agree. Other solution would be making FSImplPtr FS::_impl public or global static instance of SPIFFSImpl. Currently we are unable to call gc() because SPIFFSImpl* _fs is unreachable. What would you recommend?

Reply:
Ok, since NO_GLOBAL_SPIFFS is available option, I will create my spiffs instance locally. Removing this pull request.